### ISSUE TITLE
Setting unknown property: craft\commerce\models\PaymentSource::userId 

### ISSUE THREAD

**Issue by leevigraham** [craftcms/commerce-stripe#201]
### Description

Making payment using Stripe throws error: `Setting unknown property: craft\commerce\models\PaymentSource::userId `

### Steps to reproduce

1. Setup site using latest betas
2. Use default templates
3. Use Stripe as payment gateway
4. Checkout

![Screen Shot 2022-04-20 at 9 48 45 am](https://user-images.githubusercontent.com/25124/164119244-79b5c78f-2a9c-4978-b8d2-6f55a67a8b49.png)

Stacktrace:

```
[2022-04-19 23:44:35] web.ERROR: craft\commerce\errors\PaymentSourceException: Setting unknown property: craft\commerce\models\PaymentSource::userId in /usr/local/var/www/craft4.localhost/vendor/craftcms/commerce/src/services/PaymentSources.php:263
Stack trace:
#0 /usr/local/var/www/craft4.localhost/vendor/craftcms/commerce/src/controllers/PaymentsController.php(252): craft\commerce\services\PaymentSources->createPaymentSource(1, Object(craft\commerce\stripe\gateways\PaymentIntents), Object(craft\commerce\stripe\models\forms\payment\PaymentIntent))
#1 [internal function]: craft\commerce\controllers\PaymentsController->actionPay()
#2 /usr/local/var/www/craft4.localhost/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /usr/local/var/www/craft4.localhost/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#4 /usr/local/var/www/craft4.localhost/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('pay', Array)
#5 /usr/local/var/www/craft4.localhost/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('commerce/paymen...', Array)
#6 /usr/local/var/www/craft4.localhost/vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('commerce/paymen...', Array)
#7 /usr/local/var/www/craft4.localhost/vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#8 /usr/local/var/www/craft4.localhost/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#9 /usr/local/var/www/craft4.localhost/public/index.php(13): yii\base\Application->run()
#10 {main} {"trace":[],"memory":2557248,"category":"craft\\commerce\\errors\\PaymentSourceException","exception":"[object] (craft\\commerce\\errors\\PaymentSourceException(code: 0): Setting unknown property: craft\\commerce\\models\\PaymentSource::userId at /usr/local/var/www/craft4.localhost/vendor/craftcms/commerce/src/services/PaymentSources.php:263)"} 
[2022-04-19 23:44:35] web.INFO: Request context
```

### Additional info

- Craft CMS version: 4.0.0-beta.4
- Stripe for Craft Commerce version: 3.0.0-beta.1
- PHP version: 8.1
- Database driver & version: mariadb
- Plugins & versions:

**Comment by leevigraham** [comment]
I'm pretty sure https://github.com/craftcms/commerce-stripe/blob/develop/src/gateways/PaymentIntents.php#L282 should be:

```php
return new PaymentSource([
                'customerId' => $userId,
                'gatewayId' => $this->id,
                'token' => $stripeResponse->id,
                'response' => $stripeResponse->toArray(),
                'description' => $description,
            ]);
```

**Comment by leevigraham** [comment]
Additionally `response` must be a string… so setting it to the `$stripeRespose->toArray()` throws an error

**Comment by nfourtythree** [comment]
This issue is now resolved on the `3.0` branch. An updated beta release will be coming soon, at the same time as the next Commerce 4 beta.

Thanks!
